backend/api/chatLink/index.ts

```diff
@@ -3,28 +3,29 @@ import express from 'express';
 import db from '../../db';
```

Comment:
Can you please change the file name as well? `chatHash` and adjust the changes in https://github.com/muke1908/chat-e2ee/blob/master/backend/api/index.ts